README.md

```diff
@@ -10,51 +10,88 @@ __The project is in its *very* early stages, there is nothing to see here yet.__
 
 Tobira consists of two parts:
 
+- `frontend` contains the web interface written in TypeScript/React.
 - `backend` contains the video portal backend written in Rust which takes care of data persistence and communication to Opencast.
-- `frontend` contains the web interface written in Typescript/React.
 
 
 ## How to build Tobira
```

Comment:
This might seem like a radical thing to say, but do we really want to document this? I would hope that no one ever has to go through this. I think we should rather have a script that produces a deployable build for those that really want to build from source. But I guess that's a different topic ...

Reply:
It is. So, let's not discuss this here.
For now, we don't have this and I actually expect that people may still want to build only one part.